pkg/cli/cmds/server.go

```diff
@@ -16,7 +16,7 @@ const (
 
 var (
 	DisableItems = []string{"rke2-coredns", "rke2-ingress-nginx", "rke2-kube-proxy", "rke2-metrics-server"}
-	CNIItems     = []string{"canal", "cilium"}
+	CNIItems     = []string{"canal", "cilium", "calico"}
```

Comment:
Lets keep this sorted.
```suggestion
	CNIItems     = []string{"calico", "canal", "cilium"}
```

Reply:
done